<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2000</width>
    <height>2000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;QScrollArea&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;QScrollArea&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <widget class="QPushButton" name="grayButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Afficher masque N&amp;B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TraitementButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>ðŸŽ¯Afficher les contours</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectedImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>ðŸ“‚ Charger une image</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>281</width>
      <height>201</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="title">
     <string>ParamÃ¨tres de traitement</string>
    </property>
    <widget class="QSlider" name="sliderKsize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="maximum">
      <number>21</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TickPosition::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelKsizeValue">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>taille de noyau du filtre</string>
     </property>
    </widget>
    <widget class="QLabel" name="ksize_value">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>37</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QSlider" name="sliderLowThreshold">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>70</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TickPosition::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelLowValue">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>seuil bas de Canny</string>
     </property>
    </widget>
    <widget class="QLabel" name="low_value">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>37</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QSlider" name="sliderHighThreshold">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>180</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TickPosition::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelHighValue">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>seuil haut de Canny</string>
     </property>
    </widget>
    <widget class="QLabel" name="high_value">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>160</y>
       <width>37</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="ClickableLabel" name="labelImage_contours">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>300</y>
      <width>341</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string>                                        &quot;Image avec contours ici&quot;</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="ClickableLabel" name="labelImage">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>351</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string>                                                  &quot;Image originale ici&quot;</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="ClickableLabel" name="labelImage_Mask">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>351</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string>                        &quot;Image avec masque ici&quot;</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveMaskButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>310</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ðŸ’¾Enregistrer </string>
    </property>
   </widget>
   <widget class="QPushButton" name="SavecontoursButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>610</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ðŸ’¾Enregistrer </string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>grayButton</zorder>
   <zorder>TraitementButton</zorder>
   <zorder>SelectedImage</zorder>
   <zorder>labelImage_contours</zorder>
   <zorder>labelImage</zorder>
   <zorder>labelImage_Mask</zorder>
   <zorder>SaveMaskButton</zorder>
   <zorder>SavecontoursButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2000</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
